CHANGELOG.md

```diff
@@ -0,0 +1,98 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+## [Unreleased]
+
+NOTICE: The ABI has changed, please make sure to update your nwnx.nss and recompile all your scripts. 
+
+### Added
+- New plugins: Area, CombatModes, Effect, Encounter, Feedback, Rename, Visibility
```

Comment:
Probably better to break the list into one line per each, with a short description.